cli/net.go

```diff
@@ -416,6 +420,10 @@ var NetFindPeer = &cli.Command{
 
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
+		}
```

Comment:
You should be doing these checks first up in the command, it's the cheapest check to perform and it's good to avoid unnecessary work if you're just going to error out after doing it. Note here in particular you're doing it _after_ we call `GetFullNodeAPI` which sets up a network connection to the full node and instantiates a full client wrapper. Not super expensive but if you're just going to error then we should avoid it. So arg checks first.

But then, if you notice, at the top there is already a check, it just isn't handled properly. In particular it should return an error so the command has a non-zero exit code at least. You could move your check up and use a `IncorrectNumArgsWithHint` saying `"must provide a peerID"`.